docs/index.html

```diff
@@ -4,6 +4,7 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Carr's Computer Consulting</title>
+    <script src="./main.js"></script>
```

Comment:
You don't need this script here unless you're wanting to verify the form on this page as well, in which case you need to add the `contactForm` id to this form too.